src/components/App/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Layout from '../Layout/Layout';
+import Header from '../Header/Header';
+import Home from '../../pages/Home/Home';
+import Footer from '../Footer/Footer';
+import { BrowserRouter } from 'react-router-dom';
+
+// import styles from './App.module.css';
+
+function App() {
+    return (
+        <BrowserRouter>
+            <Header></Header>
```

Comment:
If the component does not have children it can be called using only the _open_ tag
```suggestion
            <Header />
```